### PR TITLE
Update discovery-ontology.ttl

### PR DESCRIPTION
**Pull request by AndreaCimminoArriaga**
Ontology updated with WotDirectory, WotThing types (subclasses of did:Service)

### CODE REVIEW

**Comment by mmccool** [comment]
You didn't join last meeting and I realized I should have added some comments.  We need a separate ontology file for this, not changes to the current one.  I don't want the DID file to have to pull in all the other definitions, and don't want to have to change the URL or content in DID documents from what's currently in the PR against the DID documents.

**Comment by mmccool** [comment]
It's possible I don't understand what you did, though... if you can't join the meetings, can you explain in comments here?

**Comment by AndreaCimminoArriaga** [comment]
The ontology contains the definitions of all the concepts related to the wot discovery, as far as I know this is the best way to proceed.  I created two new types discovery:WotThing and discovery:WotDirectory both subclasses of did:Service (this is how the did document specifies it should be extended with new type of services). 

On the other hand, in order to not need to change anything in the json-ld payloads the key is to have separate context files (that's why PR #512 creates a different context file). Following this approach, the discovery (not did) payloads may keep using the context as they are doing now (no need to change anything), the did payloads can use (as they do now) the WotThing and WotDirectory types (so anything needs to be changed). The only thing to take into account is that the did payloads must use the context file from PR #512.

File: context/discovery-ontology.ttl [hunk 1]
```diff
@@ -105,6 +106,18 @@ discovery:ThingDirectory rdf:type owl:Class ;
 discovery:ThingCollection rdf:type owl:Class ;
 	rdfs:label "ThingCollection".
 
+### discovery:WotThing
+discovery:WotThing rdf:type owl:Class ;
```
Comment (mmccool) [inline context/discovery-ontology.ttl:110]:
```suggestion
discovery:WotThingDIDService rdf:type owl:Class ;

```

File: context/discovery-ontology.ttl [hunk 2]
```diff
@@ -105,6 +106,18 @@ discovery:ThingDirectory rdf:type owl:Class ;
...
 	rdfs:label "ThingCollection".
 
+### discovery:WotThing
+discovery:WotThing rdf:type owl:Class ;
+	rdfs:comment "Class for DID documents";
+	rdfs:subClassOf did:Service ;
+	rdfs:label "WotThing".
+
+### discovery:WotDirectory
+discovery:WotDirectory rdf:type owl:Class ;
```
Comment (mmccool) [inline context/discovery-ontology.ttl:116]:
```suggestion
discovery:WotDirectoryDIDService rdf:type owl:Class ;

```

File: context/discovery-ontology.ttl [hunk 3]
```diff
@@ -105,6 +106,18 @@ discovery:ThingDirectory rdf:type owl:Class ;
...
+	rdfs:comment "Class for DID documents";
```
Comment (mmccool) [inline context/discovery-ontology.ttl:117]:
```suggestion
	rdfs:comment "Class for DID documents that labels URL for service returning a TD that must be a TD Directory";

```

File: context/discovery-ontology.ttl [hunk 4]
```diff
@@ -105,6 +106,18 @@ discovery:ThingDirectory rdf:type owl:Class ;
 discovery:ThingCollection rdf:type owl:Class ;
 	rdfs:label "ThingCollection".
 
+### discovery:WotThing
+discovery:WotThing rdf:type owl:Class ;
+	rdfs:comment "Class for DID documents";
```
Comment (mmccool) [inline context/discovery-ontology.ttl:111]:
```suggestion
	rdfs:comment "Class for DID documents that labels URL for a service returning a TD";

```

File: context/discovery-ontology.ttl [hunk 5]
```diff
@@ -105,6 +106,18 @@ discovery:ThingDirectory rdf:type owl:Class ;
...
+discovery:WotThing rdf:type owl:Class ;
+	rdfs:comment "Class for DID documents";
+	rdfs:subClassOf did:Service ;
+	rdfs:label "WotThing".
+
+### discovery:WotDirectory
+discovery:WotDirectory rdf:type owl:Class ;
+	rdfs:comment "Class for DID documents";
+        rdfs:subClassOf did:Service ;
+	rdfs:label "WotDirectory".
```
Comment (mmccool) [inline context/discovery-ontology.ttl:119]:
```suggestion
	rdfs:label "WotDirectoryDIDService".

```

**Pull request merged**